templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -173,6 +173,15 @@ contents:
           connection.autoconnect-priority 100 802-3-ethernet.mtu ${iface_mtu} ${extra_phys_args}
       fi
 
+      # Update connections with master property set to use the new device name
+      for conn_uuid in $(nmcli -g UUID connection show) ; do
+        if [ "$(nmcli -g connection.master connection show uuid "$conn_uuid")" != "$old_conn" ]; then
+          continue
+        fi
+        new_device=$(nmcli --get-values connection.interface-name conn show ${conn_uuid})
```

Comment:
Shouldn't this be getting the device name from $old_conn rather than $conn_uuid?

Reply:
or maybe just use $iface

Reply:
It needs to be from ovs-if-phys0 I think and this line should be moved outside of the for loop so we only do it one time

Reply:
$iface it seems to be set only for bond and vlan I will use ovs-if-phys0 that return the proper device name
```
sh-4.4# new_device=$(nmcli --get-values connection.interface-name conn show ovs-if-phys0)
sh-4.4# echo $new_device
ens4
```